dev/planning-example.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# ============================
+# Observation Planning Toolbox
+# ============================
+"""
+The observation planning toolbox consists of a number of entities that
+simplify the planning and scheduling of observations by abstracting much
+of the boilerplate code necessary to accomplish the same thing using
+core astropy or ephemerides objects.  These can be thought of as common
+idioms or convenience classes and functions that would be recreated over
+and over by astronomers and or software engineers building telescope
+scheduling software.
+
+Important assumptions:
+
+- It should be convenient to specify dates and times in the local timezone
+  of the observer.
+
+- It should be convenient to do timezone conversions easily.
+
+  This means methods that can take dates can either take astropy
+  dates (as used internally) or python datetime objects (not naive, but
+  tagged with timezone).  So dates can be passed in as timezone-aware
+  datetime objects and results can be easily converted between such.
+
+"""
+
+# ================
+# Observer objects
+# ================
+"""
+An observer defines a single terrestrial location and applicable
+environmental attributes thereof that affect observation calculations
+at that location.
+"""
+# Define an observer by full specification
+#
+# Because there may be many attributes for an observer, I suggest using
+# keyword parameters for most of them.
+from astroplan.entity import Observer
+
+obs = Observer(name='Subaru Telescope',
+               longitude='-155:28:48.900',
+               latitude='+19:49:42.600',
+               elevation=4163,
+               pressure=615,
+               relative_humidity=0.11,
+               temperature=0,
+               timezone='US/Hawaii',
+               description="Subaru Telescope on Mauna Kea, Hawaii")
+
+# longitude and latitude would be represented internally by astropy
+# angles.Longitude/Latitude.  So any value that can naturally initialize
+# one of those objects could be passed as the parameter to Observer
+# Other items are represented internally using the appropriate astropy
+# Quantities and Units and so can be initialized similarly.
+
+# It's also possible to initialize using astropy objects and other
+# direct entities
+from astropy.coordinates.angles import Longitude, Latitude
+import astropy.units as u
+import pytz
+
+obs = entity.Observer(name='Keck 1',
```

Comment:
good point - actually I suppose this wouldn't work at all given that `entity` is not imported anywhere above.

This is the sort of thing that usually gets sorted out when the API gets turned into a test, but certainly it doesn't hurt to get it right the first time!